lib/index.ts

```diff
@@ -146,6 +149,35 @@ async function dispatchCommand(
         }
     }
 }
+const handleFloorPriceChannel = async (client: any, magicEdenData: any)  => {
```

Comment:
```suggestion
async function handleFloorPriceChannel(client: any, magicEdenData: any) {
```

No real reason to use an arrow function vs regular function here, i prefer top level functions being regular funcs